libs/cli/src/main/java/org/opensearch/cli/KeystoreConstants.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cli;
+
+public class KeystoreConstants {
```

Comment:
I don't think we need it, and even if we need it - shouldn't be part of KeyStoreWrapper or Keystore CLI?

Reply:
Moving it somewhere "neutral" was my suggestion above. 

It's used by both the Keystore CLI (as part of `KeyStoreAwareCommand`) and by the server (in the `Bootstrap` class).

Reply:
Oh I see, may be move this constant to `EnvironmentAwareCommand` ? Not really the ideal, but `KeystoreConstants` does not belong here (I think)

Reply:
Added to `EnvironmentAwareCommand`. Removed `KeystoreConstants` and `KeystoreConstantsTests`.